### ISSUE TITLE
Color Thresholds

### ISSUE THREAD

**Issue by Spankythemusical** [kalkih/mini-graph-card#101]
Home Assistant: 0.93.2
mini-graph-card-bundle: 0.4.2

I'm experiencing an odd issue and cannot reach a solution.  I have 2 graphs, one temp and one humidity.  The graphs are collecting the information, but not displaying it the same.  

Here is my Lovelace config:

```
- cards:
          - color_thresholds:
              - color: '#2dc937'
                value: 60
              - color: '#e7b416'
                value: 65
              - color: '#cc3232'
                value: 70
            entities:
              - entity: sensor.dht11_temperature
            icon: 'mdi:temperature-fahrenheit'
            name: Master Bedroom
            hours_to_show: 24
            points_per_hour: 4
            show:
              fill: false
              labels: true
              icon_adaptive_color: true
              name_adaptive_color: true
            show_indicator: false
            type: 'custom:mini-graph-card'
          - color_thresholds:
              - color: '#2dc937'
                value: 20
              - color: '#e7b416'
                value: 35
              - color: '#cc3232'
                value: 40
            entities:
              - entity: sensor.dht11_humidity
            icon: 'mdi:water-percent'
            name: Master Bedroom Humidity
            hours_to_show: 24
            points_per_hour: 4
            show:
              fill: false
              labels: true
              icon_adaptive_color: true
              name_adaptive_color: true
            show_indicator: false
            type: 'custom:mini-graph-card'
        type: vertical-stack
```

With the exception of the name, sensor, and thresholds, they are identical.  Here's what it actually shows (note the color of the name and icon):

![image](https://user-images.githubusercontent.com/38551704/58614693-08cd3800-82b1-11e9-9712-7adbd54b0c12.png)

![image](https://user-images.githubusercontent.com/38551704/58614520-7fb60100-82b0-11e9-8b1f-42d1c0823871.png)

I have tried changing the thresholds, but still experienced the same issue.

This show the same via Chrome, Firefox, and both the HA ios app as well as the Home Assistant Companion ios app.

**Comment by kalkih** [comment]
Hey, should be fixed in [v0.4.3-beta2](https://github.com/kalkih/mini-graph-card/releases/tag/v0.4.3-beta2)

**Comment by Spankythemusical** [comment]
Upgraded to the beta version and the issue disappeared.  Thank you!!!
